test.js

```diff
@@ -68,6 +68,14 @@ describe('chalk.enabled', function () {
 	});
 });
 
+describe('chalk.constructor', function () {
+	it('should create a isolated context where colors can be disabled', function () {
+		var ctx = new chalk.constructor(false);
```

Comment:
Agreed, that looks pretty ugly. But I do like being able to create the instance fully configured in one go.

`new chalk.constructor({enabled: false})` is perhaps cleaner